### DISCUSSION TITLE
filter-lang=cql2-json

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#810] [API] [answered]
In an early version of ldproxy you could specify filter-lang=cql-json. For cql2 and ldproxy 3.2.4, filter-lang=**cql2-text** is the only option. Any time plan for addning **cql2-json**?

**Comment by cportele** [comment, marked as answer]
"cql2-json" is supported again in the current master branch (after we updated the implementation to the current CQL2 JSON Schema). It will be included in the upcoming 3.3.0 release.
